src/04-test-class/index.test.ts

```diff
@@ -1,44 +1,84 @@
 // Uncomment the code below and write your tests
-// import { getBankAccount } from '.';
+import {
+  getBankAccount,
+  BankAccount,
+  InsufficientFundsError,
+  SynchronizationFailedError,
+} from '.';
 
 describe('BankAccount', () => {
+  let sut: BankAccount;
+  let intial: number;
+  let amount: number;
+  let appropriate_amount: number;
+
+  beforeEach(() => {
+    intial = 100;
+    sut = getBankAccount(intial);
+    amount = 150;
+    appropriate_amount: 50;
```

Comment:
```suggestion
  let sut: BankAccount;
  let intial = 100;
  let amount = 150;
  let appropriate_amount = 50;

  beforeEach(() => {
    sut = getBankAccount(intial);
```

Reply:
Thank you for carefully and thoroughly checking the work and making suggestions!